src/ucp/core/ucp_mm.c

```diff
@@ -988,7 +988,7 @@ void ucp_memh_put_slow(ucp_context_h context, ucp_mem_h memh)
 
 ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh)
 {
-    if (ucp_memh_is_zero_length(memh)) {
+    if (NULL == memh || ucp_memh_is_zero_length(memh)) {
```

Comment:
```suggestion
    if ((NULL == memh) || ucp_memh_is_zero_length(memh)) {
```

BTW, is this situation possible? `(memh == NULL)`